packages/fx-core/src/component/driver/script/scriptDriver.ts

```diff
@@ -25,6 +25,9 @@ export class ScriptDriver implements StepDriver {
   @hooks([addStartAndEndTelemetry(ACTION_NAME, TelemetryConstant.SCRIPT_COMPONENT)])
   async run(args: unknown, context: DriverContext): Promise<Result<Map<string, string>, FxError>> {
     const typedArgs = args as ScriptDriverArgs;
+    await context.progressBar?.next(
+      `Run command ${typedArgs.run} at ${typedArgs.workingDirectory}`
```

Comment:
Can you localize this string?

Reply:
> LGTM, please also remove unused strings in `package.nls.json`.

Some of the strings are still needed by the v2 version.